cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -2040,6 +2040,14 @@
     </description>
   </property>
 
+  <property>
+    <name>messaging.http.compress.payload</name>
+    <value>true</value>
```

Comment:
If this is enabled by default, would it affect latency to publish message using `ClientMessagingService`?

Reply:
As per any compression, it is a trade off between network bandwidth and cpu. Generally compression over network is a more desirable trade off and can usually reduce latency since much less bytes are needed to be transmitted over the network